### PR TITLE
Add test_pr_locally url to contributing.rst

### PR DESCRIPTION
**Pull request by aminvakil**
Please notice renaming `test_pr_quick_start_guide.rst` to `test_pr_locally_guide.rst` too.

### CODE REVIEW

**Comment by aminvakil** [comment]
I don't know where we have put a link to `test_pr_locally_guide` / `test_pr_quick_start_guide` :)
If you agree with this change, I will fix c.g.

File: contributing.rst [hunk 1]
```diff
@@ -22,7 +22,7 @@ Open pull requests
 ==================
 
 Look through the currently open pull requests in the collection repository.
-You can help by reviewing them. Reviews help move pull requests to merge state. Some good pull requests cannot be merged only due to a lack of reviews. For more information how to provide a good review, refer to the `review checklist <review_checklist.rst>`_.
+You can help by reviewing them. Reviews help move pull requests to merge state. Some good pull requests cannot be merged only due to a lack of reviews. For more information how to provide a good review, refer to the `review checklist <review_checklist.rst>`_. Also if you want a test a pull request locally, refer to the `test a PR locally <test_pr_locally_guide.rst>`.
```
Comment (Andersson007) [inline contributing.rst:25]:
```suggestion
You can help by reviewing them. Reviews help move pull requests to merge state. Some good pull requests cannot be merged only due to a lack of reviews. For more information how to provide a good review, refer to the `review checklist <review_checklist.rst>`_. Also if you want to test a pull request locally, refer to the `test PR guide <test_pr_locally_guide.rst>`.
```

**Comment by Andersson007** [comment]
I like the idea of renaming

**Review by Andersson007 — APPROVED** [review]
LGTM

**Comment by Andersson007** [comment]
cc @felixfontein

**Comment by aminvakil** [comment]
> I like the idea of renaming

I'll open a PR in c.g. once this got merged then, thanks!

File: contributing.rst [hunk 2]
```diff
@@ -22,7 +22,7 @@ Open pull requests
 ==================
 
 Look through the currently open pull requests in the collection repository.
-You can help by reviewing them. Reviews help move pull requests to merge state. Some good pull requests cannot be merged only due to a lack of reviews. For more information how to provide a good review, refer to the `review checklist <review_checklist.rst>`_.
+You can help by reviewing them. Reviews help move pull requests to merge state. Some good pull requests cannot be merged only due to a lack of reviews. For more information how to provide a good review, refer to the `review checklist <review_checklist.rst>`_. Also if you want to test a pull request locally, refer to the `test PR guide <test_pr_locally_guide.rst>`.
```
Comment (felixfontein) [inline contributing.rst:25]:
```suggestion
You can help by reviewing them. Reviews help move pull requests to merge state. Some good pull requests cannot be merged only due to a lack of reviews. For more information how to provide a good review, refer to the `review checklist <review_checklist.rst>`_. Also if you want to test a pull request locally, refer to the `test PR locally guide <test_pr_locally_guide.rst>`.
```

Comment (Andersson007) [reply contributing.rst:25]: I asked Amin to remove "locally" (too many "locally" IMO) but this variant is also fine:)

Comment (felixfontein) [reply contributing.rst:25]: Well, it's the title of that document ;-) Maybe we should remove the `locally` before the comma in this sentence, so we can use the correct title here?

Comment (Andersson007) [reply contributing.rst:25]: Sounds good to me:)

Comment (Andersson007) [reply contributing.rst:25]: @felixfontein could you please adjust the proposed changes?

Comment (felixfontein) [reply contributing.rst:25]: @Andersson007 done

**Comment by felixfontein** [comment]
This needs rebasing.

**Comment by aminvakil** [comment]
> This needs rebasing.

Done.

**Review by Andersson007 — APPROVED** [review]
LGTM

**Review by felixfontein — APPROVED** [review]

**Comment by Andersson007** [comment]
@aminvakil thanks for working on this!
@felixfontein thanks for reviewing!

**Pull request merged**